doc/api/globals.md

```diff
@@ -91,11 +91,15 @@ ac.abort();
 ```
 
 The `AbortController` with which the `AbortSignal` is associated will only
-ever trigger the `'abort'` event once. Any event listeners attached to the
-`AbortSignal` *should* use the `{ once: true }` option (or, if using the
-`EventEmitter` APIs to attach a listener, use the `once()` method) to ensure
-that the event listener is removed as soon as the `'abort'` event is handled.
-Failure to do so may result in memory leaks.
+ever trigger the `'abort'` event once. It is recommended that code check
+that the `abortSignal.aborted` attribute is `false` before adding an `'abort'`
```

Comment:
```suggestion
ever trigger the `'abort'` event once. It is recommended to code check
that the `abortSignal.aborted` attribute is `false` before adding an `'abort'`
```

Reply:
Microsoft style guide encourages "we recommend" instead of "it is recommended":

```suggestion
ever trigger the `'abort'` event once. We recommended that code check
that the `abortSignal.aborted` attribute is `false` before adding an `'abort'`
```

Reply:
I don't think @trivikr's suggestion is correct unless they also meant to remove the word `code` or something like that?